core/templates/domain/collection/CollectionRightsObjectFactorySpec.ts

```diff
@@ -171,11 +160,8 @@ describe('Collection rights object factory', () => {
     var sampleCollectionRights = collectionRightsObjectFactory.create(
       noUnpublishCollectionRightsBackendObject);
 
-    var emptyCollectionRightsBackendObject = (
-      collectionRightsObjectFactory.createEmptyCollectionRights());
-
-    emptyCollectionRightsBackendObject.copyFromCollectionRights(
-      sampleCollectionRights);
+    var emptyCollectionRightsBackendObject =
```

Comment:
Please wrap multiline statements in parenthesis break after `(` (see style [guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript)).

Reply:
done!